rasa/validator.py

```diff
@@ -382,7 +376,11 @@ def verify_slot_mappings(self) -> bool:
                     form_slots = self.domain.forms.get(condition_active_loop, {}).get(
                         REQUIRED_SLOTS_KEY, {}
                     )
-                    if form_slots and slot.name not in form_slots:
+                    if (
+                        form_slots
+                        and slot.name not in form_slots
+                        and mapping_type != SlotMappingType.FROM_TRIGGER_INTENT
+                    ):
```

Comment:
This is irrelevant, because it's not necessary to add the slot with `FROM_TRIGGER_INTENT` mapping to the form's `required_slots` (since it gets filled automatically at form activation without it being requested by the form) and at the same time we want to allow a `FROM_TRIGGER_INTENT` filled slot to specify in which form context they should be filled via a mapping condition.

While this validation check is meant for any other slot mapping type that has mapping conditions specified and should be part of the form`s `required_slots`, it's not applicable for this `FROM_TRIGGER_INTENT` mapping.